message_definitions/v1.0/common.xml

```diff
@@ -856,6 +856,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="188" name="MAV_CMD_DO_LAND_REJOIN" hasLocation="true" isDestination="false">
+        <description>Mission command to perform a landing. This is used as a marker in a mission to tell the autopilot that is can rejoin a mission at any point after this marker and before a landing or MAV_CMD_DO_LAND_START, the autopilot will then rejoin the mission at the closet point following it to the landing.</description>
+        <param index="1">Empty</param>
+        <param index="2">Empty</param>
+        <param index="3">Empty</param>
+        <param index="4">Empty</param>
+        <param index="5" label="Latitude">Latitude</param>
+        <param index="6" label="Longitude">Longitude</param>
+        <param index="7">Empty</param>
```

Comment:
why no altitude?

Reply:
Not sure, is just the same as DO_LAND_START

Reply:
now has alt